atest/robot/standard_libraries/process/process_library.robot

```diff
@@ -25,3 +25,6 @@ Input things to process
 
 Get process id
     Check Test Case    ${TESTNAME}
+
+Assign process object to variable
+    Check Test Case    ${TESTNAME}
```

Comment:
Newline missing at the end of file. Good idea to configure your IDE to add them automatically.

This test should actually be before Get process id to keep them in same order as in the test data side.

Reply:
moved the test case